airbyte-db/lib/build.gradle

```diff
@@ -69,3 +72,29 @@ task(dumpJobsSchema, dependsOn: 'classes', type: JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
     args 'jobs', 'dump_schema'
 }
+
+task copyDocker(type: Copy) {
+    delete "build/docker"
+
+    from "${project.projectDir}/Dockerfile"
+    into "build/docker/"
+}
+
+task copyGeneratedXml(type: Copy) {
```

Comment:
Is this an accurate name since it also includes the sql file? Would it make sense to call this `copyResources`?